css/index.css

```
@@ -104,14 +104,117 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
+    background: teal;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.middle-content .boxes .box1 {
+    width: 12.5%;
+    height: 100px;
+    background: gold;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.middle-content .boxes .box2 {
+    width: 12.5%;
+    height: 100px;
+    background:cadetblue;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.middle-content .boxes .box3 {
+    width: 12.5%;
+    height: 100px;
+    background: coral;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.middle-content .boxes .box4 {
+    width: 12.5%;
+    height: 100px;
+    background: crimson;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.middle-content .boxes .box5 {
+    width: 12.5%;
+    height: 100px;
+    background: forestgreen;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.middle-content .boxes .box6 {
+    width: 12.5%;
+    height: 100px;
+    background:darkorchid;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.middle-content .boxes .box7 {
+    width: 12.5%;
+    height: 100px;
+    background: hotpink;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.middle-content .boxes .box8 {
+    width: 12.5%;
+    height: 100px;
+    background: indigo;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.middle-content .boxes .box9 {
+    width: 12.5%;
+    height: 100px;
+    background: dodgerblue;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
```

We can trim this code down by quite a bit by doing this.
First, we give all the boxes their size and positioning:
```suggestion
	.middle-content .boxes .box {
    width: 12.5%;
    height: 100px;
    margin: 20px 2.5%;
    color: white;
    display: flex;
    align-items: center;
    justify-content: center;
}
```
Then we can give them some color:

```suggestion
.middle-content .boxes .box1 {
  background: teal;   
}

.middle-content .boxes .box2 {
  background: gold;   
}

.middle-content .boxes .box3 {
  background: cadetblue;   
}

.middle-content .boxes .box4 {
  background: coral;   
}

.middle-content .boxes .box5 {
  background: crimson;   
}

.middle-content .boxes .box6 {
  background: forestgreen;   
}

.middle-content .boxes .box7 {
  background: darkorchid;   
}

.middle-content .boxes .box8 {
  background: hotpink;   
}

.middle-content .boxes .box9 {
  background: indigo;   
}

.middle-content .boxes .box10 {
  background: dodgerblue;   
}
```